src/nameTable.java

```diff
@@ -3,21 +3,38 @@
 import src.parseTree.tokens.id;
 
 import java.util.HashMap;
+import java.util.Map;
 
 public class nameTable {
```

Comment:
nit: since this follows a singleton pattern, if class name reflected that, it would be a bit easier to recognize. not a huge deal though

Reply:
Done